src/Text/Literate.idr

```diff
@@ -55,7 +59,7 @@ rawTokens : (delims  : List (String, String))
 rawTokens delims ls =
           map (\(l,r) => (block l r, CodeBlock (trim l) (trim r))) delims
        ++ map (\m => (line m, CodeLine (trim m))) ls
-       ++ [(any, Any)]
+       ++ [(notCodeLine, Any)]
 
 ||| Merge the tokens into a single source file.
 reduce : List (TokenData Token) -> String -> String
```

Comment:
Clarifying here: 

+ Suspicious in what way?
+ How would `List String` accumulator be better? 
+ I was not aware that there was a `fastAppend` in `libs`?

Reply:
Regardless, I do kinda see where you are coming from. The clarification request is so that *I* actually understand your point. I have updated the PR with the proposed suggestions.

Reply:
The fact we are appending *on the right* may not be a problem despite
what my intuition was telling me: it's strings we are dealing with, not
lists.

But the fact we are constantly appending (thus creating a lot of
intermediate strings, by copying the content of each pairs of buffer
into one large enough to hold both strings) may very well be.

With `fastAppend` we collect the parts and only at the end do we bother
generating a new string big enough to hold the content of all of the parts.